cli/cmd/top.go

```diff
@@ -1,284 +1,287 @@
 package cmd
 
 import (
-	"bufio"
```

Comment:
Thanks for submitting this improvement, @kohsheen1234 ! Before we get deeper into the review, it appears that all of the whitespace in this file (cli/cmd/top.go) was changed, so it's difficult to see what logic changed, if any. As far as I can tell, there's probably no need for this file to change in this PR at all? Can you revert the unnecessary changes to this file?  Thanks!

Reply:
Sure @olix0r . Sorry for the inconvenience , those filed are changed because of my previous pull request which isn't merged yet. I will surely make the changes as required as soon as possible :)

Reply:
Hey @olix0r , For reviewing the changes you can check out this link. 

https://github.com/linkerd/linkerd2/pull/4013/files#diff-a15e4fe04e9a7d4097aed3e6a41db024L658

It contains all the changes I have made for this issue. Can you can pull my latest changes and test it locally, then I will create a formal pull request again :)

Reply:
Thanks! much clearer now ;)

We're working on a release at the moment, but someone will get to reviewing this soon--hopefully in the next few days.

Thanks again for contributing to Linkerd!